### PR TITLE
GOINGTOSAI: SWI-4528 - Prepare saiport.py file for upstreaming

### PR DESCRIPTION
**Pull request by MariuszStachura**

### CODE REVIEW

File: ptf/saiport.py [hunk 1]
```diff
@@ -0,0 +1,2829 @@
...
+            # Remove the qos_map
+            status = sai_thrift_set_port_attribute(
+                self.client, self.portx,
+                qos_pfc_priority_to_priority_group_map=0)
+            status = sai_thrift_remove_qos_map(self.client, qos_map_id)
+            status = sai_thrift_remove_port(self.client, self.portx)
+
+
+@group("draft")
+class SAIBridgePortNoFloodTest(SaiHelperBase):
```
Comment (ravi861) [inline ptf/saiport.py]: Shouldn't this move to saibridge_port.py

File: ptf/saiport.py [hunk 2]
```diff
@@ -0,0 +1,2829 @@
...
+                                eth_src=src_mac,
+                                pktlen=100)
+        print("Sending packet on port with no bridge port created")
+        send_packet(self, self.dev_port1, pkt)
+        verify_no_other_packets(self)
+        print("\tPacket dropped. OK")
+
+
+@group("draft")
+class SAIBridgePortFloodClass(SaiHelperBase):
```
Comment (ravi861) [inline ptf/saiport.py]: same here

**Review by ravi861 — APPROVED** [review]
LGTM. Ok to upstream

File: ptf/saiport.py (hunk 1 above)
Comment (aljer) [reply ptf/saiport.py]: I agree, we should move both of them. I have already PR for bridgeport module opened in OCP repo. I will add these tests to it so please remove them from this file.

**Review by aljer — APPROVED** [review]
In general LGTM, just please use imperative forms in docstrings (e.g. 'test' instead of 'tests', 'verify' instead of 'verifies'

File: ptf/saiport.py (hunk 2 above)
Comment (MariuszStachura) [reply ptf/saiport.py]: SAIBridgePortFloodClass is used as a base class for PortIngressMirrorSessionTest, PortEgressMirrorSessionTest and PortAclBindingClass, I renamed it to PortFloodClass

File: ptf/saiport.py (hunk 1 above)
Comment (MariuszStachura) [reply ptf/saiport.py]: Thanks, addressed in next commit.

**Pull request closed without merging**